package.json

```diff
@@ -23,12 +24,13 @@
   "devDependencies": {
     "eslint-config-agreeto": "workspace:*",
     "eslint": "^8.26.0",
-    "husky": "^4.3.8",
+    "husky": "^8.0.1",
```

Comment:
updated the husky version so you don't have to do the manual chmod "hack"

Reply:
Nice! How did I end up using v4 when 8 is already out? haha

Does that also mean we can add `lint-staged` again?